src/main/java/com/amazon/ion/impl/IonReaderContinuableApplicationBinary.java

```diff
@@ -45,9 +45,19 @@ class IonReaderContinuableApplicationBinary extends IonReaderContinuableCoreBina
 
     // The UTF-8 encoded bytes representing the text `$ion_symbol_table`.
     private static final byte[] ION_SYMBOL_TABLE_UTF8;
+    private static final byte[] IMPORTS_UTF8;
+    private static final byte[] SYMBOLS_UTF8;
+    private static final byte[] NAME_UTF8;
+    private static final byte[] VERSION_UTF8;
+    private static final byte[] MAX_ID_UTF8;
 
     static {
         ION_SYMBOL_TABLE_UTF8 = SystemSymbols.ION_SYMBOL_TABLE.getBytes(StandardCharsets.UTF_8);
+        IMPORTS_UTF8 = SystemSymbols.IMPORTS.getBytes(StandardCharsets.UTF_8);
+        SYMBOLS_UTF8 = SystemSymbols.SYMBOLS.getBytes(StandardCharsets.UTF_8);
+        NAME_UTF8 = SystemSymbols.NAME.getBytes(StandardCharsets.UTF_8);
+        VERSION_UTF8 = SystemSymbols.VERSION.getBytes(StandardCharsets.UTF_8);
+        MAX_ID_UTF8 = SystemSymbols.MAX_ID.getBytes(StandardCharsets.UTF_8);
     }
```

Comment:
Minor suggestion to reduce the total number of LOC.

I know it wraps it the GitHub UI, but it's only 120 characters in the longest case if we put the declaration and initialization on one line.

```suggestion
    private static final byte[] ION_SYMBOL_TABLE_UTF8 = SystemSymbols.ION_SYMBOL_TABLE.getBytes(StandardCharsets.UTF_8);
    private static final byte[] IMPORTS_UTF8 = SystemSymbols.IMPORTS.getBytes(StandardCharsets.UTF_8);
    private static final byte[] SYMBOLS_UTF8 = SystemSymbols.SYMBOLS.getBytes(StandardCharsets.UTF_8);
    private static final byte[] NAME_UTF8 = SystemSymbols.NAME.getBytes(StandardCharsets.UTF_8);
    private static final byte[] VERSION_UTF8 = SystemSymbols.VERSION.getBytes(StandardCharsets.UTF_8);
    private static final byte[] MAX_ID_UTF8 = SystemSymbols.MAX_ID.getBytes(StandardCharsets.UTF_8);
```

Reply:
Done